save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -112,13 +112,13 @@ fun ChildrenBuilder.fileUploaderForSandbox(
     fileUploaderOverSandboxFileInfo {
         isSandboxMode = true
         selectedFiles = selectedFilesFromState
-        getUrlForAvailableFilesFetch = { "$sandboxApiUrl/list-file?userName=$userName" }
-        getUrlForFileUpload = { "$sandboxApiUrl/upload-file?userName=$userName" }
+        getUrlForAvailableFilesFetch = { "$sandboxApiUrl/list-file" }
+        getUrlForFileUpload = { "$sandboxApiUrl/upload-file" }
         getUrlForFileDownload = { fileInfo ->
-            "$sandboxApiUrl/download-file?userName=$userName&fileName=${fileInfo.name}"
+            "$sandboxApiUrl/download-file?fileName=${fileInfo.name}"
```

Comment:
Here and below: consider a file named `&foo=bar`. Such a name will be interpreted as an extra request parameter. File names need to be escaped first.

Reply:
Don't really want to add dependency on `ktor`, because of it, but make sense, added